### PR TITLE
[14.0] [MIG] stock_demand_estimate_matrix

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v.14. Adding "ir.model.access.csv".

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@DavidBForgeFlow Could you squash the last 2 commits and check runbot warnings?

File: stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+            "src_model": "stock.demand.estimate.wizard",
+            "view_type": "form",
+            "view_mode": "tree",
+            "res_model": "stock.demand.estimate",
+            "type": "ir.actions.act_window",
+        }
+        return res
+
+
+class StockDemandEstimateSheetLine(models.TransientModel):
```
Comment (rousseldenis) [inline stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py]: Please put one model per file

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. LGTM apart the file split

**Comment by rousseldenis** [comment]
@DavidBForgeFlow Could you check travis?

File: stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py]: so today it is a blocking request? :laughing:

Comment (LoisRForgeFlow) [reply stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py]: I don't want to sound rude, it is just that this is not a big deal and will add a big diff in the migration as well, but we will do it now and we will correct the commit history as well.

Thanks for the time you spend reviewing.

Comment (DavidBForgeFlow) [reply stock_demand_estimate_matrix/wizards/stock_demand_estimate_wizard.py]: Done!

**Review by MateuGForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Works perfectly, tested on local.

**Pull request merged**